### PR TITLE
In-Store Pickup GraphQl intersection products assignments filter.

### PR DESCRIPTION
**Pull request by swnsma**
## Purpose of this pull request

Fixes https://github.com/magento/devdocs/issues/7701

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.4/graphql/queries/pickup-locations.html

## Links to Magento source code
magento/inventory#3161

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by keharper** [comment]
@swnsma This appears to be a 2.4.1 addition. If so, I'll need you to either rebase this PR to 2.4.1-develop or open a new PR against that branch. If it's questionable whether this code will make it into the 2.4.1, you might need to wait until we create a 2.4.2-develop branch.

**Comment by keharper** [comment]
This didn't make it into 2.4.1. Retagged to 2.4.2.

**Comment by swnsma** [comment]
@keharper 
Related PR to source code has been merged.

**Comment by swnsma** [comment]
Hi @keharper !
Any future for this PR?

**Comment by keharper** [comment]
@swnsma I don't understand the `ProductInfoInput` object. Is it being used to search for locations that have all the listed SKUs or that do not have the listed SKUs?

**Comment by swnsma** [comment]
Hi @keharper,

It will leave only locations which have relations with listed SKU.
If at least one SKU is missing in location, location will not be added to the result list.

File: src/guides/v2.4/graphql/queries/pickup-locations.md [hunk 1]
```diff
@@ -117,6 +122,7 @@ Attribute | Data type | Description
 `sort` | [PickupLocationSortInput](#PickupLocationSortInput) | Specifies which attribute to sort on, and whether to return the results in ascending or descending order.
 `pageSize` | Int | The maximum number of pickup locations to return at once. The attribute is optional. The default value is 20.
 `currentPage` | Int | Specifies which page of results to return. The default value is 1.
+`productsInfo` |[[ProductInfoInput]](#ProductInfoInput)| Information about products which should be delivered. The information is used to filter Pickup Locations which do not have relations with mentioned products.
```
Comment (keharper) [inline src/guides/v2.4/graphql/queries/pickup-locations.md:125]:
```suggestion
`productsInfo` |[[ProductInfoInput]](#ProductInfoInput)| Contains an array of SKUs to filter on.
```

**Review by keharper — CHANGES_REQUESTED** [review]
@swnsma  I think the note on lines 17-18 needs to be deleted.  I suspect that `productsInfo` or `sku` should be added to the bulleted list at line 14. If yes, then use a variant of my suggestion for line 176.

**Comment by keharper** [comment]
@swnsma can you take a look at this PR again?

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**